app/models/group.rb

```diff
@@ -604,4 +604,12 @@ def limit_inheritance
   def self.with_one_coordinator
     published.select{ |g| g.admins.count == 1 }
   end
+
+  def default_cover_photo
+    if self.default_group_cover
+      /^.*(?=\?)/.match(self.default_group_cover.cover_photo.url).to_s
```

Comment:
After a little experimentation, this is too slow to do in a migration, I think. The paperclip transfer is slow (~2/3 sec per record), and we've got over 20,000 groups in prod with no cover photo.

```
Group.joins('LEFT JOIN groups p ON p.id = groups.parent_id')
     .where('groups.cover_photo_file_name IS NULL AND p.cover_photo_file_name IS NULL')
     .count
=> 22220
```

That spins out to like a 7 hour migration.

We could do something like 'only groups with no cover photo created in the last 90 days' or something, which cuts down the numbers significantly, but then we've gotta keep the spare code in there anyway and I'm tempted not to bother.